test/helpers.js

```diff
@@ -53,17 +47,14 @@ export function tryToReadFile(file) {
 	}
 }
 
+const { window } = new JSDOM('<main></main>');
+global.document = window.document;
```

Comment:
I think that's meant as security advice, for cases where you're dealing with untrusted content (and can't risk giving access to the filesystem, for example). It would probably be a bit of a pain to get the tests to work with `window.eval`, since they need to be able to import other components, and we need to override `requestAnimationFrame` etc.